src/components/ui4/ButtonsViewUi4.tsx

```diff
@@ -19,6 +20,8 @@ export interface ButtonInfo {
   disabled?: boolean
   spinner?: boolean
   testID?: string
+
+  animProps?: EdgeAnimProps
```

Comment:
Optional: I don't especially like this, because it's too low level. We don't let the user tweak other button properties at this level of detail. Could it be something simpler, like an `animate: boolean` prop on the top-level `ButtonsViewUi4` component? Then the buttons view is in charge of the visuals again, not the scene.

I don't think we should fix this now, because we are running out of time, but at least start thinking about other solutions.

Reply:
I agree, it would be nice to be simpler. 

It's like this because each scene has its own timing depending on the number of items animating, I couldn't figure out how to get around that beyond giving animation handling to the SceneWrapper, and the SceneWrapper takes chunks of components like SectionView does.

Reply:
Maybe instead we give `ButtonsViewUi4` an `animStartDistance` time?

Reply:
Yeah, something like this would be good.